pkg/server/container_start.go

```diff
@@ -315,23 +396,39 @@ func setOCILinuxResource(g *generate.Generator, resources *runtime.LinuxContaine
 }
 
 // setOCICapabilities adds/drops process capabilities.
-func setOCICapabilities(g *generate.Generator, capabilities *runtime.Capability) error {
-	if capabilities == nil {
-		return nil
-	}
+func setOCICapabilities(g *generate.Generator, capabilities *runtime.Capability, privileged bool) error {
+	if privileged {
+		// Add all capabilities in privileged mode.
+		var finalCapList []string
+		for _, cap := range capability.List() {
+			if g.HostSpecific && cap > validate.LastCap() {
+				continue
+			}
+			finalCapList = append(finalCapList, fmt.Sprintf("CAP_%s", strings.ToUpper(cap.String())))
+		}
+		spec := g.Spec()
+		spec.Process.Capabilities.Bounding = finalCapList
+		spec.Process.Capabilities.Effective = finalCapList
+		spec.Process.Capabilities.Inheritable = finalCapList
+		spec.Process.Capabilities.Permitted = finalCapList
+		spec.Process.Capabilities.Ambient = finalCapList
+	} else {
+		if capabilities == nil {
+			return nil
+		}
 
-	for _, c := range capabilities.GetAddCapabilities() {
-		if err := g.AddProcessCapability(c); err != nil {
-			return err
+		for _, c := range capabilities.GetAddCapabilities() {
+			if err := g.AddProcessCapability(c); err != nil {
```

Comment:
We need to do `fmt.Sprintf("CAP_%s", strings.ToUpper(cap.String()))` for the CAP passed via CRI.
We don't have the `CAP_` prefix in Kubernetes/CRI.